Query/ResolvedSelectQuery.php

```diff
@@ -242,13 +230,11 @@ public function getSearchTerm(): string
     /**
      * {@inheritdoc}
      */
-    public function getFacets()
+    public function getFacets(): array
     {
-        if ($this->selectQuery->getFacets()) {
-            return $this->selectQuery->getFacets();
-        }
-
-        return $this->searchContext->getDefaultFacets();
+        $facets = $this->selectQuery->getFacets();
+        
+        return $facets ?? $this->searchContext->getDefaultFacets();
```

Comment:
Hello bug. :)
"??" is not equivalent of the previous code, "?:" is. Resulted in facets not being returned by solr query at all. Will fix in next release.

cc: @calumbrodie